src/ngAria/aria.js

```diff
@@ -226,7 +235,7 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
           }
         },
         post: function(scope, elem, attr, ngModel) {
-          var needsTabIndex = shouldAttachAttr('tabindex', 'tabindex', elem);
+          var needsTabIndex = shouldAttachAttr('tabindex', 'tabindex', elem, nodeBlackList);
```

Comment:
`shouldAttachAttr()` does not take `nodeBlackList` into account. I think ti should...
(Either take it into account or remove this 4th argument - to avoid confusion.)